torch_xla/__init__.py

```diff
@@ -1,10 +1,22 @@
 import logging
 import os
 import re
+import sys
 import tempfile
 import warnings
 
 import torch
+
+if not torch.cuda.is_available():
+  # Load _XLAC_cuda_functions to RTLD_GLOBAL, so that it can be used by _XLAC.
```

Comment:
can you point me to the place where _XLAC_cuda_functions is used by _XLAC?

Reply:
oh I guess you mean the phone definition will be made available when we do `import _XLAC` below

Reply:
Yes. Exactly.